includes/view/User_view.php

```diff
@@ -641,13 +643,17 @@ function User_view(
                                 form_hidden('user', $user_source->id),
                                 form_submit('submit', __('arrived'), '', false, 'primary')
                             ], page_link_to('admin_arrive'), true) : '',
-                        $admin_user_privilege ? button(
-                            page_link_to(
-                                'users',
-                                ['action' => 'edit_vouchers', 'user_id' => $user_source->id]
-                            ),
-                            icon('file-binary-fill') . __('Edit vouchers')
-                        ) : '',
+                        $admin_user_privilege ? 
```

Comment:
Using ` $admin_user_privilege && config('enable_voucher') ? [...]` would be easier :)

Reply:
done